src/main/java/gov/usgs/wma/waterdata/DiscreteGroundWaterRowMapper.java

```diff
@@ -10,14 +10,14 @@
 	@Override
 	public DiscreteGroundWater mapRow(ResultSet rs, int rowNum) throws SQLException {
 
-		DateTimeAccuracy dta = DateTimeAccuracy.parse(rs.getString("field_visit_comments"));
+		DiscreteGroundWaterRules rules = new DiscreteGroundWaterRules();
 
 		DiscreteGroundWater discreteGroundWater = new DiscreteGroundWater();
 		discreteGroundWater.setFieldVisitIdentifier(rs.getString("field_visit_identifier"));
 		discreteGroundWater.setLocationIdentifier(rs.getString("location_identifier"));
 		discreteGroundWater.setAgencyCode(rs.getString("agency_code"));
-		discreteGroundWater.setDateTimeAccuracyCode(dta.getCode());
-		discreteGroundWater.setDateTimeAccuracyText(dta.getText());
```

Comment:
Previously, there was some business logic in here, parsing the comment field and assigning the dta code and text based on it.  Now that has all moved to the 'rules' class and these two derived fields are left as null until the rules are applied.